tests/ddos/ddo_sample1_v4.py

```diff
@@ -5,11 +5,11 @@
 json_dict = {
     "@context": ["https://w3id.org/did/v1"],
     "id": "did:op:0c184915b07b44c888d468be85a9b28253e80070e5294b1aaed81c2f0264e430",
-    "created": "2000-10-31T01:30:00.000-05:00",
-    "updated": "2000-10-31T01:30:00.000-05:00",
-    "version": "v4.0.0",
+    "version": "4.0.0",
     "chainId": 1337,
     "metadata": {
+        "created": "2000-10-31T01:30:00.000-05:00",
+        "updated": "2000-10-31T01:30:00.000-05:00",
```

Comment:
In the latest DDO changes, it looks like they added the timezone `Z` postfix back in.